analysis/combiners.py

```diff
@@ -216,31 +230,35 @@ class SumCombiner(UtilityAnalysisCombiner):
     AccumulatorType = Tuple[float, float, float, float, float]
 
     def __init__(self, params: pipeline_dp.combiners.CombinerParams):
-        self._params = params
+        self._params = copy.copy(params)
 
-    def create_accumulator(self, data: PreaggregatedData) -> AccumulatorType:
+    def create_accumulator(
+            self, data: Tuple[np.ndarray, np.ndarray,
+                              np.ndarray]) -> AccumulatorType:
         count, partition_sum, n_partitions = data
+        del count  # not used for SumCombiner
```

Comment:
Can't we do `_, partition_sum, n_partitions = data` instead in Python? (also in other places)

Reply:
I see. IMHO when you do `_, partition_sum, n_partitions = data` you don't even think about `count`, so there is a smaller cognitive load while reading the code; so I would prefer that. But I definitely see the benefit of being explicit. So, no strong opinions.